src/components/FlowRunTimeline.vue

```diff
@@ -292,6 +292,7 @@
     })
 
     expandedSubFlowRuns.value.set(id, {
+      // @ts-expect-error data actually is a valid property
```

Comment:
Did this error not go away when you were not linked? There shouldn't be a typescript error.

Reply:
I forgot to `npm ci` after resetting `package.json`. Yeah it's not complaining now.